docs/python/index.asciidoc

```diff
@@ -7,7 +7,26 @@ ground for all Elasticsearch-related code in Python; because of this it tries
 to be opinion-free and very extendable. The full documentation is available at
 http://elasticsearch-py.rtfd.org/
 
-It can be installed with:
```

Comment:
This may get lost in the text. How about adding a title (maybe with [float]?) or a sidebar to make it stand out more:

    .Elasticsearch DSL
    **************
    Your text here
    **************